com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -703,22 +661,27 @@ private void ClearParameters()
         }
 
         /// <summary>
-        /// Shutdown
         /// Flushes the internal messages
         /// Removes itself from the network update loop
         /// Disposes readers, writers, clears the queue history, and resets any parameters
         /// </summary>
-        public void Shutdown()
+        private void Shutdown()
         {
+            if (NetworkLog.CurrentLogLevel == LogLevel.Developer)
+            {
+                NetworkLog.LogInfo($"[Instance : {s_RpcQueueContainerInstances}] {nameof(RpcQueueContainer)} shutting down.");
+            }
+
             //As long as this instance is using the pre-defined update stages
             if (!m_ProcessUpdateStagesExternally)
             {
                 //Remove ourself from the network loop update system
                 this.UnregisterAllNetworkUpdates();
             }
 
+            bool isListening = !ReferenceEquals(m_NetworkManager, null) && m_NetworkManager.IsListening;
```

Comment:
Again, I think we can be watertight and not need these null checks (which don't show up everywhere)

Reply:
Actually, that is there specifically for a bug that isn't pertaining to the RpcQueueContainer itself but was happening.  During shutdown there were instances where the NetworkManager was not finishing its shutdown sequence but would have its Singleton gone.  This makes sure that we don't dive into the RpcQueueProcessor to flush any remaining queue items and try to send out to the transport when there is no transport to send out on.

Reply:
Fine, but I am going to ask you to polish & hoist the comment you just wrote to me into the code.  This is a good subtlety to capture.  Otherwise it looked like to me that it's a normal condition for this to be null, we just aren't always checking.

Reply:
Actually, I think I am going to just set that value to false and skip that check.  
You are right... if m_NetworkManager is null then something is very wrong.